src/components/Header/HeaderUser.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from "react"
+import { useSelector, useDispatch } from "react-redux"
+import { NavLink, useLocation } from 'react-router-dom'
+
+import { MAIN_DARK, AUTH_TEXT_COLOR } from "../../assets/themes/colors"
+
+import { fetchLogout } from "../../ducks/userInfo"
+import LogoHead from "./LogoHead"
+import PointsBal from "./PointsBal"
+import MobileMenu from "./MobileMenu"
+import Logouticon from "./Logouticon"
+import Navigation from "./Navigation"
+import {
+    DesktopWidth,
+    TabletWidth,
+    MobileWidth,
+    ContainerHeader,
+    LeftBlock,
+    MenuHeaderBlock,
+    MenuContact,
+    MenuSection,
+    MobileHamburgerContainer,
+    MobileHamburgerLine,
+    MobileHamburgerDarkened,
+    MobileHamburgerbackground
+} from "./Header.styled"
+
+const linkActiveColorDesktop = ({ isActive }) => { return { color: isActive ? MAIN_DARK : AUTH_TEXT_COLOR } }
+
+const HeaderUser = () => {
+    const dispatch = useDispatch()
+    const logout = () => dispatch(fetchLogout(formData))
```

Comment:
Looks like you didnt define formData var and we get an error into console that could be probably a crash)
![image](https://user-images.githubusercontent.com/52242019/189525835-4c0154f2-6bf7-4d5a-85fe-bac58cef7fdf.png)
PS you didnt need it at all